### DISCUSSION TITLE
lib FILE not found

### DISCUSSION THREAD

**Discussion by Asyedabdulrahman** [Cyfrin/foundry-full-course-cu#1021] [Q&A]
![Screenshot (5)](https://github.com/Cyfrin/foundry-full-course-f23/assets/110740098/51a21b2a-05e3-4aff-a1fb-f1040485e27a)
while passing command(forge init) except ** lib file** all are downloading. can someone share the solution how to solve it .even though i created a new file, then also its not working.

 
![Screenshot (5)](https://github.com/Cyfrin/foundry-full-course-f23/assets/110740098/ed955428-ddd8-42cd-92fd-0787b5329ffe)

**Comment by cromewar** [comment]
Hey @Asyedabdulrahman can you please more specific with the question please?

**Comment by olahamid** [comment]
> ![Screenshot (5)](https://user-images.githubusercontent.com/110740098/275620239-51a21b2a-05e3-4aff-a1fb-f1040485e27a.png) while passing command(forge init) except ** lib file** all are downloading. can someone share the solution how to solve it .even though i created a new file, then also its not working.
> 
> ![Screenshot (5)](https://user-images.githubusercontent.com/110740098/275620991-ed955428-ddd8-42cd-92fd-0787b5329ffe.png)

if you havn't written toom much in you current deployraffle contract. you can delete the folder and run forge init --force command again.  or you can create a new file run the command and move the lib folder in the raffle foder.
